docs/content/en/docs/user-guide/event-watcher.md

```diff
@@ -0,0 +1,152 @@
+While it empowers you to build pretty versatile workflows, the canonical use case is that you trigger a new deployment by image updates, package releases, etc.
+
+This guide walks you through configuring Event watcher and how to push an Event.
+
+## Prerequisites
+Before configuring EventWatcher, be sure to grant write permission to the SSH key used by Piped. See [here](/docs/operator-manual/piped/configuring-event-watcher/) for more details.
+
+## Usage
+File updating can be done by registering the latest value corresponding to the Event in the control-plane and comparing it with the current value.
+
+Therefore, you mainly need to:
+1. define which values in which files should be updated when a new Event found.
+1. integrate a step to push an Event to the control-plane using `pipectl` into your CI workflow.
+
+### 1. Define Events
+Prepare EventWatcher files placed at the `.pipe/` directory at the root of the Git repository.
+In that files, you define which values in which files should be updated when the Piped found out a new Event.
+
+For instance, suppose you want to update the Kubernetes manifest defined in `helloworld/deployment.yaml` when an Event with the name `helloworld-image-update` occurs:
+
+```yaml
+apiVersion: pipecd.dev/v1beta1
+kind: EventWatcher
+spec:
+  events:
+    - name: helloworld-image-update
+      replacements:
+        - file: helloworld/deployment.yaml
+          yamlField: $.spec.template.spec.containers[0].image
+```
+
+The full list of configurable `EventWatcher` fields are [here](/docs/user-guide/configuration-reference/#event-watcher-configuration).
+
+### 2. Pushing an Event with `pipectl`
+To register a new value corresponding to Event such as the above in the control-plane, you need to perform `pipectl`.
+And if you want to push Events continuously, we highly recommend integrating a step for that into your CI workflow.
+
+According to [this guide](http://localhost:1313/docs/user-guide/command-line-tool/#installation), you first install it on your CI system or where you want to run.
+And then you grab the API key for the control-plane according to [this guide](http://localhost:1313/docs/user-guide/command-line-tool/#authentication).
+
+Once you're all set up, now all we need to do is to execute a command like:
+
+```bash
+pipectl event register \
+    --address=CONTROL_PLANE_API_ADDRESS \
+    --api-key=API_KEY \
+    --name=helloworld-image-update \
+    --data=gcr.io/pipecd/helloworld:v0.2.0
+```
+
+
+After a while, Piped will create a commit as shown below:
+
+```diff
+     spec:
+       containers:
+       - name: helloworld
+-        image: gcr.io/pipecd/helloworld:v0.1.0
++        image: gcr.io/pipecd/helloworld:v0.2.0
+```
+
+NOTE: Keep in mind that it will take a little while because Piped checks if a new Event exists by periodically issuing a request to the control-plane. You can set its interval according to [here](/docs/operator-manual/piped/configuring-event-watcher/#optional-settings-for-watcher).
```

Comment:
"Keep in mind that it may take a little while because Piped periodically fetches the new events from the control-plane check."
"You can change its interval"